human-formatter/index.js

```diff
@@ -203,6 +215,7 @@ class HumanFormatter extends stream.Writable {
       let note = record.note
       if (typeof note === 'string') {
         note = note.replace(/`([^`]+)`/g, c.bold('$1'))
+        // TODO: Why concat with spreading?
         note = [].concat(
```

Comment:
It could be a bug after ES5 → ES6 migration. Try to remove it.

Reply:
Hm, spread operator is ES6, so I suppose this code is already ES6. 
I was just confused that such complex way had been selected. 
I'd prefer to write it with `flatMap`
`note.split('\n').flatMap(row => splitByLength(row, 80 - PADDING.length))`

Reply:
What about `flatMap` or it's better to stick to ES5 solution?

Reply:
Yeap, `flatMap` fits perfectly here.